whois-commons/src/main/java/net/ripe/db/whois/common/DatabaseDummifierJmx.java

```diff
@@ -64,6 +68,9 @@ public class DatabaseDummifierJmx extends JmxBase {
     private static final AtomicInteger jobsAdded = new AtomicInteger();
     private static final AtomicInteger jobsDone = new AtomicInteger();
 
+    private static final Set<String> AVAILABLE_ENVIRONMENTS = ImmutableSet.of("dev", "prepdev", "test", "rc",
+            "training");
+
```

Comment:
If you are introucing Environment , can you create an enum for it?

Reply:
There is also sentry.environment , can you replace it with this new property?

Reply:
as we talked in standup this is a jmx script, we are specifying the properties in the input. We are not taking the properties from the whois property file. Look the database info input.